--- src/list.c.ORIG	2020-11-25 21:23:40.623260931 +0100
+++ src/list.c	2020-11-25 21:23:45.855301827 +0100
@@ -44,9 +44,11 @@
     if(list->last == NULL) {
         list->first = node;
         list->last = node;
+node->prev = NULL;
     } else {
         list->last->next = node;
         node->prev = list->last;
+node->next = NULL;
         list->last = node;
     }
 
@@ -102,8 +102,10 @@
         list->last = NULL;
     } else if(node == list->first) {
         list->first = node->next;
+list->first->prev = NULL;
     } else if (node == list->last) {
         list->last = node->prev;
+list->last->next = NULL;
     } else {
         ListNode *after = node->next;
         ListNode *before = node->prev;
