--- src/ini.c.orig	2018-12-26 21:08:15.289767000 +0000
+++ src/ini.c	2018-12-26 21:07:25.707778000 +0000
@@ -187,6 +187,17 @@
                 value = lskip(value);
                 rstrip(value);
 
+                /* Strip surrounding double and single quotes */
+                if ((*value == '"') && (value[strlen(value) - 1] == '"')) {
+                    value[strlen(value) - 1] = '\0';
+                    value += 1;
+                } else {
+                    if ((*value == '"') && (value[strlen(value) - 1] == '"')) {
+                        value[strlen(value) - 1] = '\0';
+                        value += 1;
+                    }
+                }
+
                 /* Valid name[=:]value pair found, call handler */
                 strncpy0(prev_name, name, sizeof(prev_name));
                 if (!HANDLER(user, section, name, value) && !error)
