--- src/ini.c.ORIG	2020-11-12 23:40:31.773490733 +0100
+++ src/ini.c	2020-11-12 23:41:06.825555329 +0100
@@ -187,6 +187,17 @@
                 value = lskip(value);
                 rstrip(value);
 
+                /* Strip surrounding double and single quotes */
+                if ((*value == '"') && (value[strlen(value) - 1] == '"')) {
+                    value[strlen(value) - 1] = '\0';
+                    value += 1;
+                } else {
+                    if ((*value == '\'') && (value[strlen(value) - 1] == '\'')) {
+                        value[strlen(value) - 1] = '\0';
+                        value += 1;
+                    }
+                }
+
                 /* Valid name[=:]value pair found, call handler */
                 strncpy0(prev_name, name, sizeof(prev_name));
                 if (!HANDLER(user, section, name, value) && !error)
