--- src/ini.h.ORIG	2020-11-26 03:37:52.533640594 +0100
+++ src/ini.h	2020-11-26 03:38:43.553738846 +0100
@@ -14,6 +14,13 @@
 #ifndef INI_H
 #define INI_H
 
+/* START For Snoopy - redefine function names, to avoid symbol name collisions */
+#define ini_parse          snoopy_ini_parse
+#define ini_parse_file     snoopy_ini_parse_file
+#define ini_parse_stream   snoopy_ini_parse_stream
+#define ini_parse_string   snoopy_ini_parse_string
+/* END For Snoopy */
+
 /* Make this header file easier to include in C++ code */
 #ifdef __cplusplus
 extern "C" {
